.github/workflows/ci.yml

```diff
@@ -100,7 +100,8 @@ jobs:
         # that are needed during the build process. Additionally, this works
         # around a bug in the 'cache' action that causes directories outside of
         # the workspace dir to be saved/restored incorrectly.
-        run: echo "::set-env name=CARGO_HOME::$(pwd)/.cargo_home"
+        shell: pwsh
```

Comment:
Why change the shell from default?

Reply:
linux and windows diff

if use bash in windows `CARGO_HOME: /d/a/deno/deno/.cargo_home`

Reply:
`pwsh` exists in all os